swagger/swagger-generator/generator-core/src/main/java/io/servicecomb/swagger/generator/core/OperationGenerator.java

```diff
@@ -221,6 +221,24 @@ protected void scanMethodAnnotation() {
       }
       processor.process(annotation, this);
     }
+
+    setDefaultTag();
+  }
+
+  private void setDefaultTag() {
+    // if tag has been defined, do nothing
+    if (null != operation.getTags()) {
```

Comment:
never can get a null value?

Reply:
this modify has been rolled back. users determine what will be printed in these fields by themselves.